python_modules/libraries/dagster-dbt/dagster_dbt/utils.py

```diff
@@ -310,3 +310,13 @@ def get_dbt_resource_props_by_dbt_unique_id_from_manifest(
         **manifest["metrics"],
         **manifest.get("semantic_models", {}),
     }
+
+
+def _set_flag_attr(key, value):
```

Comment:
nit, so we don't have to iterate through all of the global flags twice
```suggestion
def _set_flags(kvs: Sequence[Tuple[str, Any]]):
```